sections/advanced/email-testing.md

```diff
@@ -0,0 +1,213 @@
+# Email Testing
+
+<br/><br/>
+
+### One Paragraph Explainer
+
+Email testing [critical for business success](https://www.industrialmarketer.com/why-email-testing-is-critical-for-email-marketing-success/) and [boosts email performance](https://litmus.com/blog/3-reasons-why-email-testing-boosts-email-performance). 
+It is not something we want to forego while testing our web applications because modern email services allow painless automated email testing.
+Typically email testing involves validating email fields (from, to, cc, subject, attachments), html content and links in the email. Email services also allow spam checks and visual checks.
+The core goal is to enable the last mile of end to end testing, to enable a typical webapp to be test from start to finish.
+
+That being stated, email testing is a fundamental enabler for SaaS test architectures by enabling stateless tests that can scale; tests that independently handle their state and can be executed by n number of entities at the same time. 
+Check out the topic [**Test States**](./sections/advanced/test-states.md) for further discussion on the topic.
+
+<br/><br/>
+
+
+## Foreword
+
+If you are using [gmail tricks](https://www.idownloadblog.com/2018/12/19/gmail-email-address-tricks/) or [AWS Simple Email Service](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/send-email-simulator.html) and these use cases are satisfactory for your test needs without any side effects, only topic 1 might be of interest to you.
```

Comment:
I’m not 100% sure what is the "topic 1" you are referencing to at the end of this paragraph

Reply:
converted to an in-page link